### ISSUE TITLE
Sound Effect decreasing in volume has sudden spike in volume

### ISSUE THREAD

**Issue by michaelphipps** [processing/p5.js-sound#158]
Hi, I've been pointed here from the p5.js community forum.

Background: I'm making a simple bouncing ball that is affected by gravity. Every time the ball hits the bottom of the screen, it plays a bounce sound. It also decreases the volume slightly on every bounce.

I have an issue where there are sudden spikes in the volume of the sound.  Also, I'm perplexed by the fact the sound does not start off at maximum volume, even though I set that volume in setup().

I've recorded a video to explain the problem further in some detail, showing some of the things I have tried. 
https://youtu.be/KIxTjinyNHk

I've also got my current source code here: https://github.com/michaelphipps/p5js-bouncy-ball

I thought this was a problem in my code, until I was pointed to the issue queue, I didn't even consider it might be a problem in the library!  It's still most likely it is my code.  

Any ideas?

**Comment by michaelphipps** [comment]
Just wondering if this may be related to issue #56

**Comment by michaelphipps** [comment]
I've been able to work around the volume spike issue by explicitly providing the time, rate and amp parameters in the play method.  Also of note is that if the volume is set to 0, .play(time, rate, amp) will play the sound full volume.  I think this is by design both so the sound plays full volume when no volume is specified, and also because there is no point to playing a sound no one can hear.  

so I've been able to resolve most the issue using the following code in my app:
```
if (this.volume > 0){
                    bounceSound.play(0,1,this.volume); //time, rate, amp
                } 
``` 
I'm still having issues with the initial sound not being played at full volume, so if anyone can offer suggestions about that, that would be really appreciated!
